packages/playground/src/components/layout/top-bar-widget.tsx

```diff
@@ -19,7 +19,7 @@ import {
 import { getInitials } from '../../utils/utils'
 
 interface Project {
-  id: string
+  id: number
   name: string
 }
```

Comment:
Don't add this new interface, instead use interface [TypesSpace](https://github.com/harness/react-api-client/blob/main/packages/code-service/src/services/schemas/TypesSpace.ts) from `@harnessio/code-service-client`

`import { TypesSpace } from '@harnessio/code-service-client'`